filebeat/module/logstash/slowlog/_meta/fields.yml

```diff
@@ -32,6 +35,7 @@
       description: >
         Execution time for the plugin in milliseconds.
     - name: plugin_params
+      # Keyword?
```

Comment:
This looks like it's an unparsed JSON object for the plain log, and it's moved (and parsed) to `.plugin_params_object` in the JSON log pipeline.

I'd once again follow the convention and have as keyword or non-indexed for now, and if need be add .text as a multi-field.

Reply:
Changed to `keyword` with `text` multi-field in https://github.com/elastic/beats/pull/10417.